www/js/diary/list.js

```diff
@@ -42,6 +42,8 @@ angular.module('emission.main.diary.list',['ui-leaflet',
     // TODO: Convert the usercache calls into promises so that we don't have to
     // do this juggling
     Timeline.updateForDay(day);
+    // This will be used to show the date of datePicker in the user language.
+    $scope.currDay = moment(day).format('DD MMM YYYY');
```

Comment:
again, I would suggest using `LL` or `ll`. The default string format in the US is `MMM DD YYYY` not `DD MMM YYYY`. As somebody who moved to the US from India, I had to get used to the change as well :)